pkg/securitycontextconstraints/sccadmission/admission.go

```diff
@@ -495,16 +500,43 @@ func shouldIgnore(a admission.Attributes) (bool, error) {
 	if pod.Spec.OS != nil && pod.Spec.OS.Name == coreapi.Windows {
 		return true, nil
 	}
-	// if this is an update, see if we are only updating the ownerRef.  Garbage collection does this
-	// and we should allow it in general, since you had the power to update and the power to delete.
-	// The worst that happens is that you delete something, but you aren't controlling the privileged object itself
-	if a.GetOperation() == admission.Update && rbacregistry.IsOnlyMutatingGCFields(a.GetObject(), a.GetOldObject(), kapihelper.Semantic) {
-		return true, nil
+
+	if a.GetOperation() == admission.Update {
+		oldPod, ok := a.GetOldObject().(*coreapi.Pod)
+		if !ok {
+			return false, admission.NewForbidden(a, fmt.Errorf("object was marked as kind pod but was unable to be converted: %v", a.GetOldObject()))
+		}
+
+		// never ignore any spec changes
+		if !kapihelper.Semantic.DeepEqual(pod.Spec, oldPod.Spec) {
+			return false, nil
+		}
+
+		// see if we are only doing meta changes that should be ignored during admission
+		// for example, the OVN controller adds informative networking annotations that shouldn't cause the pod to go through admission again
+		if shouldIgnoreMetaChanges(pod.ObjectMeta, oldPod.ObjectMeta) {
+			return true, nil
+		}
 	}
 
 	return false, nil
 }
 
+func shouldIgnoreMetaChanges(newMeta, oldMeta metav1.ObjectMeta) bool {
+	// check if we're changing only annotations from the ignore list or not
+	for a, newVal := range newMeta.Annotations {
```

Comment:
what if I remove an annotation?

Reply:
Implemeted this case and added respective unit tests :+1:

Reply:
could you please `s/a/key/`?